addons/mail/static/src/discuss/call/common/rtc_session_model.js

```diff
@@ -197,6 +197,9 @@ export class RtcSession extends Record {
         }
         try {
             await this.audioElement.play();
+            if (this.store.settings.audioOutputDeviceId) {
+                await this.audioElement.setSinkId(this.store.settings.audioOutputDeviceId);
+            }
```

Comment:
Couldn't this be moved outside of try/catch?
It's better to minimise LOCs that are prone to error and/or catch should be silent on very specific issue. Current `catch()` is silent on any error therefore should be moved outside of it or change the `catch()` to be more specific. I think the 1st approach of move outside is easier.

Reply:
I extracted it and just added a `.catch()` to the promise as it could throw if the permission to the output device is removed, or if the device is no longer available (which can be silent, it will use the default output).